staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,62 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called `.import-restrictions`. If this file is not found, parent directories will be recursively searched.
+  - NOTE: Adding `.import-restrictions` files does not add them to CI runs. In order for them to be added, they need to be explicitly added to `arguments.InputDirs`. `InputDirs` is defined within `import-boss/main.go`.
+
+  Here's an example code snippet of how it would look:
+  ```go
+  arguments.InputDirs = []string{
+    "k8s.io/kubernetes/pkg/...",
+    "k8s.io/kubernetes/cmd/...",
+    "k8s.io/kubernetes/plugin/...",
+  }
+  ```
+
+- If an `.import-restrictions` file is found, then all imports of the package are checked against each rule in the file. A rule consists of three parts:
+  - A `SelectorRegexp`, to select the import paths that the rule applies to.
+  - A list of `AllowedPrefixes`
+  - A list of `ForbiddenPrefixes`
+
+- An import is allowed if it matches at least one allowed prefix and does not match any forbidden prefixes. An example `.import-restrictions` file looks like this:
+
+```json
+{
+  "Rules": [
+    {
+      "SelectorRegexp": "k8s[.]io",
+      "AllowedPrefixes": [
+        "k8s.io/gengo/examples",
+        "k8s.io/kubernetes/third_party"
+      ],
+      "ForbiddenPrefixes": [
+        "k8s.io/kubernetes/pkg/third_party/deprecated"
+      ]
+    },
+    {
+      "SelectorRegexp": "^unsafe$",
+      "AllowedPrefixes": [
+      ],
+      "ForbiddenPrefixes": [
+        ""
+      ]
+    }
+  ]
+}
+```
+- NOTE: The second block explicitly matches the unsafe package, and forbids it ("" is a prefix of everything).
```

Comment:
`the unsafe package` -> `the "unsafe" package`

i'd also explain with a sentence how `"SelectorRegexp": "k8s[.]io"` is handled above.

Reply:
possibly omit the `NOTE:` prefix again.